flask/app/models.py

```diff
@@ -9,26 +9,6 @@
 from . import db
 
 
-class User(UserMixin, db.Model):
-    user_id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(30), nullable=False, unique=True)
-    password_hash = db.Column(db.String(200), nullable=False, unique=False)
-    email = db.Column(db.String(150), nullable=False, unique=True)
-    is_admin = db.Column(db.Boolean, unique=False, default=False)
-    last_login = db.Column(db.DateTime)
-    deactivated = db.Column(db.Boolean, unique=False, nullable=False, default=False)
-
-    def set_password(self, password):
-        self.password_hash = generate_password_hash(
-            password, method="pbkdf2:sha256:50000"
-        )
-
-    def check_password(self, password):
-        return check_password_hash(self.password_hash, password)
-
-    def get_id(self):
-        return self.user_id
-
```

Comment:
Could you move the `user_groups` table up instead of moving the `User` model down?